### PR TITLE
Post summary

### PR DESCRIPTION
**Pull request by ataulm**
This is just binding all the data and click listeners.

screenshot-2015-07-12_17 52 42 509

### CODE REVIEW

File: core/src/main/java/com/ouchadam/loldr/feed/FeedActivity.java [hunk 1]
```diff
@@ -27,11 +28,12 @@
     private static final String ACTION = BuildConfig.APPLICATION_ID + ".FEED";
 
     private static final String EXTRA_SUBREDDIT = "subreddit";
-    private static final String DEFAULT_SUBREDDIT = "askreddit";
+    private static final String DEFAULT_SUBREDDIT = "news";
```
Comment (ataulm) [inline core/src/main/java/com/ouchadam/loldr/feed/FeedActivity.java]: Dunno what's the best subreddit to get a mix of self, image thumbs and text links...

File: core/src/main/java/com/ouchadam/loldr/feed/MarshallerFactory.java [hunk 2]
```diff
@@ -43,32 +43,47 @@ static MarshallerFactory newInstance(Resources resources) {
     private Marshaller<Ui.PostSummary, Data.Post> post() {
         return new Marshaller<Ui.PostSummary, Data.Post>() {
             @Override
-            public Ui.PostSummary marshall(final Data.Post from) {
+            public Ui.PostSummary marshall(final Data.Post post) {
```
Comment (ataulm) [inline core/src/main/java/com/ouchadam/loldr/feed/MarshallerFactory.java]: such a pet peeve when people use the word "from" as a var.

File: data/src/main/java/com/ouchadam/loldr/data/Data.java [hunk 3]
```diff
@@ -16,12 +16,18 @@ private Data() {
 
         String getSubreddit();
 
-        int getUps();
+        int getScore();
```
Comment (ataulm) [inline data/src/main/java/com/ouchadam/loldr/data/Data.java]: score is the one you see on reddit when there's only one value - score == ups - downs

File: core/src/main/java/com/ouchadam/loldr/feed/MarshallerFactory.java (hunk 2 above)
Comment (ouchadam) [inline core/src/main/java/com/ouchadam/loldr/feed/MarshallerFactory.java]: haha bet you hate `T what` as well!, such a @blundell checkstyle

File: data/src/main/java/com/ouchadam/loldr/data/Data.java (hunk 3 above)
Comment (ouchadam) [inline data/src/main/java/com/ouchadam/loldr/data/Data.java]: :+1:

File: core/src/main/java/com/ouchadam/loldr/feed/FeedActivity.java (hunk 1 above)
Comment (ouchadam) [inline core/src/main/java/com/ouchadam/loldr/feed/FeedActivity.java]: sounds legit

**Pull request merged**